schema/catvrs/catvrs-source.yaml

```diff
@@ -13,296 +13,254 @@ namespaces:
+          description: >-
+            MUST be a `Coding` representing one of "EFO:0030069" (complete genomic loss), "EFO:0020073" (high-level loss),"EFO:0030068" (low-level loss), "EFO:0030067" (loss), "EFO:0030064" (regional base ploidy),"EFO:0030070" (gain), "EFO:0030071" (low-level gain), "EFO:0030072" (high-level gain).
     required:
-      - location
+      - copies
+
 
-  DescribedVariation:
+
+
+  # =============================================================================
+  # Base CatVar object class
+  # =============================================================================
+
+
+  CategoricalVariant:
     maturity: draft
     type: object
-    inherits: CategoricalVariation
+    inherits: gks.common:DomainEntity
     description: >-
-      Some categorical variation concepts are supported by custom nomenclatures or text-descriptive
-      representations for which a categorical variation model does not exist. DescribedVariation is
-      a class that adds requirements and contextual semantics to the `label` and `description` fields
-      to indicate how a categorical variation concept should be evaluated for matching variants.
+      A representation of an intensionally-defined categorical domain for variation, in which individual 
+      contextual variation instances may be members of the domain.
     properties:
       type:
         extends: type
-        const: DescribedVariation
-        default: DescribedVariation
-        description: 'MUST be "DescribedVariation"'
+        const: CategoricalVariant
+        default: CategoricalVariant
+        description: 'MUST be "CategoricalVariant"'
+      id: #ref gks-common?
+        type: string
+        description: >-
+          The 'logical' identifier of the entity in the system of record, e.g. a UUID. This 'id' is unique within a given system. The identified entity may have a different 'id' in a different  system, or may refer to an 'id' for the shared concept in another system (e.g. a CURIE).
       label:
         extends: label
         description: >-
-          A primary label for the categorical variation. This required property should provide a 
-          short and descriptive textual representation of the concept.
-      description:
-        extends: description
+            A primary label for the categorical variation. This required property should provide a short and descriptive textual representation of the concept.
+      constraints:
+        type: array
+        items: #do we have to have a value here for JSON schema or just require it to be non-empty?
+      extensions:    # Are extensions still being used in current VRS? 
+        type: array
+        ordered: true
+        items:
+          $refCurie: gks.core:Extension
+        description: 
+          A list of extensions to the entity. Extensions are not expected to be natively understood, but may be used for pre-negotiated exchange of message attributes between systems.
```

Comment:
```suggestion
```
Extensions are still in use, but no need to redefine the property here unless there is a Cat-VRS-specific description or usage you intend to convey. Which doesn't appear to be the case here.